src/App.js

```diff
@@ -1,13 +1,44 @@
 import React, { Component } from 'react'
 import { Instructions } from './components'
+import { Counter } from './components'
+import { InitialCountForm } from './components'
 import './styles/app.css'
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      count: 0
+    }
+
+    this.updateCounter = this.updateCounter.bind(this)
```

Comment:
Instead of using .bind, use ES6 autobinding, which requires the use of arrow functions.

instead it would look lik
```
updateCounter(newCount) = () => {
      this.setState({ count: newCount })
}
````